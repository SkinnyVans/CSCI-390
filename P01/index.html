<!DOCTYPE html>
<html>

	<head>
		<title>Home</title>
	</head>

	<body>
		<div>
			<a href="index.html">Home</a>
			<a href="mlb.html">MLB</a>
			<a href="nfl.html">NFL</a>
		</div>
		<h1>Sports</h1>
		<h2>This is a website about our favorite sports</h2>
		<img alt="Sports Logo" src="images/mlbnfl.jpg" />
		<h2>On this site, you can find:</h2>
		<ul>
			<li>MLB News</li>
			<li>NFL News</li>
			<li>Our Favorite MLB Teams</li>
			<li>Our Favorite NFL Teams</li>
		</ul>
	</body>

</html>