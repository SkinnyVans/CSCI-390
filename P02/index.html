<!DOCTYPE html>
<html>
	<head>
		<link rel="stylesheet" href="style/style.css" />
		<meta charset="UTF-8">
		<meta name="viewport" content="width=device-width, initial-scale=1">
		<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
		<title>Home</title>
	</head>

	<body>
		<div class="w3-row">
			<?php include("menu.html"); ?>
		</div>
		<h1 class="underline">Sports</h1>
		<div class="w3-row-padding">
			<div class="w3-half">
				<p>This is a website about our favorite sports</p>
				<p>This website discusses <em>MLB</em> and <em>NFL</em> news</p>
				<img alt="Sports Logo" src="images/mlbnfl.jpg" />
			</div>
			<div class="w3-half">
				<h1>On this site, you can find:</h1>
				<ul>
					<li><strong>MLB</strong> News</li>
					<li><strong>NFL</strong> News</li>
					<li>Our Favorite MLB Teams</li>
					<li>Our Favorite NFL Teams</li>
				</ul>
			</div>
		</div>
		<div class="w3-row">
			<?php include("footer.html"); ?>
		</div>
	</body>

</html>