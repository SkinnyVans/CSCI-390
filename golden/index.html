<!DOCTYPE html>
<html>
	<head>
		<link rel="stylesheet" href="style/style.css" />
		<title>New Test Page</title>
	</head>
	<body>
		<section>
			<h1>Test</h1>
			<p>This is some test text</p>
		</section>
			<img src="img/golden.jpg" alt="golden retriever"/>
	</body>

	<footer>
		Copyright &copy; 2017 UIndy
	</footer>
</html>